setup homepage

ugwulo opened the pull request:
I have resolved the issue with the build exception error, the issue was caused because I used the new <com.google.android.material.navigation.NavigationView> instead of <fragment> in the navhost fragment.

please review @jumaallan @mikailyusuf

ugwulo committed:
setup homepage

jumaallan commented on app/src/main/java/com/example/gitrank/ui/Explore.kt:
```diff
@@ -0,0 +1,35 @@
+import com.example.gitrank.databinding.FragmentExploreBinding
+
+
+class Explore : Fragment() {
```
I think it's best practise to name the classes well. For example, if we call this `Explore`, someone might not understand it well

We can rename it to `ExploreFragment`

ugwulo replied:
My bad, 
I will sure update them

jumaallan commented on app/src/main/java/com/example/gitrank/ui/Repositories.kt:
```diff
@@ -0,0 +1,36 @@
+import com.example.gitrank.databinding.FragmentUsersBinding
+
+
+class Repositories : Fragment() {
```
Call this `RepositoriesFragment`

jumaallan commented on app/src/main/java/com/example/gitrank/ui/Users.kt:
```diff
@@ -0,0 +1,37 @@
+import com.example.gitrank.databinding.FragmentUsersBinding
+
+
+class Users : Fragment() {
```
This to `UsersFragment`

jumaallan reviewed (changes requested):
Look at the requested changes

ugwulo committed:
Rename Explore.kt to ExploreFragment.kt

ugwulo committed:
renamed:    app/src/main/java/com/example/gitrank/ui/Explore.kt -> app/src/main/java/com/example/gitrank/ui/ExploreFragment.kt
	renamed:    app/src/main/java/com/example/gitrank/ui/Repositories.kt -> app/src/main/java/com/example/gitrank/ui/RepositoriesFragment.kt
	renamed:    app/src/main/java/com/example/gitrank/ui/Users.kt -> app/src/main/java/com/example/gitrank/ui/UsersFragment.kt

ugwulo committed:
Merge branch 'develop' of https://github.com/GoogleDevXProgram/GitRank into homepage
setup kotlin DSL

ugwulo committed:
setup Gitrank navigation with bottom navigation
NavigationExtensions.kt for managing multiple naigation graphs

ugwulo committed:
resolve conflicting file {ExploreFragment.kt}

ugwulo commented:
I have used the appropriate file names for the Fragments and went on to modify my earlier commit to use the recommended bottom navigation for multiple nav graphs. Reference here -> https://github.com/android/architecture-components-samples
Please review @jumaallan @mikailyusuf 

![Screenshot_20210504-085743](https://user-images.githubusercontent.com/52312550/116982861-41f25c00-acc1-11eb-89b5-bf1877563385.png)

mikailyusuf commented:
LGTM
